CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # NHS.UK frontend Changelog
 
+## Unreleased
+
+:wrench: **Fixes**
+
+- On devices with "camera notches", browsers reserve a safe area in landscape orientation (known as pillar-boxing) so content isn't obscured.
+
+  To avoid this, support has been added for `viewport-fit=cover` as shown here:
+  https://webkit.org/blog/7929/designing-websites-for-iphone-x/
+
+  ([PR #1510](https://github.com/nhsuk/nhsuk-frontend/pull/1510))
```

Comment:
I suppose it was under "New features" over in GOV.UK Frontend

But for us, we have "Changes" which could be more appropriate?

i.e. It's not broken currently (but this is an improvement)

```suggestion
:recycle: **Changes**

We've updated our grid layout gutter margins to accommodate devices with "camera notches". Browsers on these devices reserve a safe area in landscape orientation (known as pillar-boxing) so content isn't obscured.

To avoid this, support has been added for `viewport-fit=cover` as shown here:
https://webkit.org/blog/7929/designing-websites-for-iphone-x/

This change was introduced in [pull request #1510: Accommodate camera notches](https://github.com/nhsuk/nhsuk-frontend/pull/1510).
```